experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -63,10 +63,11 @@ function ClimaLandSimulation(
     start_date::Dates.DateTime,
     output_dir::String,
     area_fraction,
-    nelements::Tuple{Int, Int} = (101, 10),
+    nelements::Tuple{Int, Int} = (101, 15),
```

Comment:
Here, would it be helpful to use the Land `global_domain` function?

Reply:
Yes! I forgot we have that now. I'll switch to using it in a separate PR so we can isolate any effects from the changes